src/drive/web/modules/actions/index.jsx

```diff
@@ -81,15 +83,15 @@ export const download = ({ client, vaultClient }) => {
           return (
             <ActionMenuItem
               onClick={props.onClick}
-              left={<Icon icon={isIOSApp() ? ShareIosIcon : ReplyIcon} />}
+              left={<Icon icon={props.icon} />}
```

Comment:
Without changing the caller, we change the result here, no?

Reply:
I have only moved the logique around `isIOSApp` inside the `actions[...].icon` field so this component will have the exact same behavior. The main difference is that the icon inside the `SelectionBar` will now have the same behaviour.